### ISSUE TITLE
Fix uninstall

### ISSUE THREAD

**Issue by djay** [collective/Products.LoginLockout#17]
- needs a test for this
- site user will see 'no attribute "_successful_login_attempts" when the plugin is uninstalled
- More of the manual install code can probably be moved to GS
- I think the configlet doesn't get removed either looking at the code.
- probably shouldn't uninstall the root level PAS plugin as that could break other sites. But what happens if you want to remove the package? Will it break the site?

**Comment by nazrulworld** [comment]
Yes we need to fix uninstall, you are right most of uninstall like tool, registry, skins, configlet could be removed by using uninstall profile. 

For uninstall plugin, we need to create Extension/install.py, In my opinion uninstall plugin is important although  as you mention `root level PAS plugin as that could break other sites`. I can share my real life experience, for some reason i had need to add extra properties inside plugin class and when I reinstall `Products.Loginlockout` but i didn't no impact (after long fight I understand that as plugin is not removed during uninstall).

As this plugin is managing internal storage, so if plugin is removed as the same time data will be removed. In that case we can add data import/export option
